.github/workflows/tests.yml

```
@@ -580,15 +580,15 @@ jobs:
       # Clean checkout of specific git ref needed for package metadata version
       # which needs env vars GIT_DESCRIBE_TAG and GIT_BUILD_STR:
       - name: Checkout Source
-        uses: actions/checkout@v4
+        uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633
```

https://github.com/actions/checkout/releases/tag/v4

```suggestion
        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
```